openstack/utils/templates/snippets/_proxysql.cfg.tpl

```diff
@@ -15,11 +15,21 @@ mysql_variables =
     {{- $monitorEnabled := false }}
     {{- $monitorUsername := "" }}
     {{- $monitorPassword := "" }}
+    {{- $monitorGalera := false }}
     {{- /*
     Check that proxysql_monitor user exists in all DB instances
     Check that proxysql_monitor credentials are the same in all DB instances
     */}}
     {{- range $dbKey, $db := $dbs }}
+        {{- if hasKey $db "pxc" }}
+            {{- $monitorEnabled = true }}
```

Comment:
Is it required to have monitoring enabled?
If not, I'd enable it conditionally, depending on whether `$db.system_users.monitor.password` exists or not.